spec/modules.md

```diff
@@ -52,9 +52,13 @@ An *empty CSS tree* contains no statements.
 ### Configuration
 
 A *configuration* is a map from [variable](variables.md) names to SassScript
-values. An *empty configuration* contains no entries.
+values. An *empty configuration* contains no entries. Each *configuration* has
+a uniquely-identifying opaque ID.
 
-[source file]: syntax.md#source-file
+Each *configuration* has a uniquely-identifying opaque ID, sharing the same
+instance across all [source files][] in a [module](#module).
```

Comment:
The first half of this is just reiterating the previous paragraph, and it's not clear exactly which new configurations the second part applies to. I'd say instead something like, "Each new configuration has a unique ID by default." Then in the text of the spec, you can say something like "Create a new configuration with the same ID as before" if you want them to share identities.

Reply:
done, I think.. :)